diff -urEb VolView-4.1.1.orig/src/config.ts VolView-4.1.1/src/config.ts
--- VolView-4.1.1.orig/src/config.ts	2023-05-19 12:58:22.916872311 +0200
+++ VolView-4.1.1/src/config.ts	2023-05-19 13:01:13.557196054 +0200
@@ -124,7 +124,7 @@
 } as Record<number, number[]>;
 
 export const SAMPLE_DATA: SampleDataset[] = [
-  {
+  /*{
     name: 'CTA Head and Neck',
     filename: 'CTA-Head_and_Neck.zip',
     description:
@@ -163,7 +163,7 @@
       '3D ultrasound of a baby. Downloaded from tomovision.com.(8 MB)',
     url: 'https://data.kitware.com/api/v1/item/635679c311dab8142820a4f4/download',
     image: USFetusThumbnail,
-  },
+  },*/
 ];
 
 export const TOOL_COLORS = [
diff -urEb VolView-4.1.1.orig/src/io/itk/itkConfig.js VolView-4.1.1/src/io/itk/itkConfig.js
--- VolView-4.1.1.orig/src/io/itk/itkConfig.js	2023-05-19 12:58:22.924872326 +0200
+++ VolView-4.1.1/src/io/itk/itkConfig.js	2023-05-19 12:58:52.900929265 +0200
@@ -1,9 +1,11 @@
 const base = process.env.VUE_APP_PUBLIC_PATH ?? '';
 const itkConfig = {
-  pipelineWorkerUrl: `${base}/itk/pipeline.worker.js`,
-  imageIOUrl: `${base}/itk/image-io`,
-  meshIOUrl: `${base}/itk/mesh-io`,
-  pipelinesUrl: `${base}/itk/pipelines`,
+  pipelineWorkerUrl: `./itk/pipeline.worker.js`,
+
+  // The URIs below are relative to "pipeline.worker.js"
+  imageIOUrl: `./image-io`,
+  meshIOUrl: `./mesh-io`,
+  pipelinesUrl: `./pipelines`,
 };
 
 export default itkConfig;
diff -urEb VolView-4.1.1.orig/vue.config.js VolView-4.1.1/vue.config.js
--- VolView-4.1.1.orig/vue.config.js	2023-05-19 12:58:22.916872311 +0200
+++ VolView-4.1.1/vue.config.js	2023-05-19 12:59:16.820974677 +0200
@@ -31,7 +31,7 @@
       return args;
     });
   },
-  publicPath: process.env.VUE_APP_PUBLIC_PATH || '/',
+  publicPath: process.env.VUE_APP_PUBLIC_PATH || '.',
   configureWebpack: {
     devtool: 'source-map',
     resolve: {
